CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 2.29.0-rc.0 / 2021-07-29
 
+Note for macOS users: Due to [changes in the upcoming Go 1.17](https://tip.golang.org/doc/go1.17#darwin),
+this is the last Prometheus release that supports macOS 10.12 Sierra and
+previous versions.
```

Comment:
This is not technically true, [Go 1.15 dropped support for anything before macOS 10.12 Sierra](https://tip.golang.org/doc/go1.15#darwin), and therefore this release is the last one to support 10.12 specifically.

This could be confusing to users who see this in the changelog and assume they can use this on their macOS <10.12, but it's not really a huge issue.

Reply:
Thanks, fixed